src/System.Private.Uri/src/System/UriFormatException.cs

```diff
@@ -2,12 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netstandard10
```

Comment:
netstandard13aot, which I had removed before but you and @tarekgh asked me to keep since it's not shipped yet, so when I added it back it broke the netstandard13aot build.